src/pages/Community/FeedDetail/Reply.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component, Fragment } from "react";
+import { BsHeart } from "react-icons/bs";
+import ReReply from "./ReReply";
+class Reply extends Component {
+  constructor() {
+    super();
+    this.state = {
+      showlist: true,
+      text: "",
+      list: [],
+    };
+  }
+  reReply = () => {
+    this.setState({ showlist: false });
+  };
+  handleInput = (e) => {
+    this.setState({ text: e.target.value });
+  };
+
+  addReReply = (e) => {
+    e.preventDefault();
+    const { text, list } = this.state;
+    if (text.length > 0) {
+      this.setState({
+        list: [...list, text],
+        text: "",
+      });
+    }
+  };
+
+  render() {
+    return (
+      <div className="Reply">
+        <li className="ReplyList">
+          <article className="ReplyContent">
+            <p className="ReplyText">
+              <a className="ReplyUserName" href="/">
+                <img
+                  className="ReplyUserProfile"
+                  src={this.props.profileImage}
+                  alt=""
+                />
+                <span className="ReplyUserId">{this.props.userId}</span>
+              </a>
+              <span className="ReplyPostText">{this.props.comment}</span>
+            </p>
+            <footer className="ReplyFooter">
+              <time className="ReplyFooterTime">몇분전</time>
+              <button className="ReplyLike">
+                <BsHeart />
+                좋아요
+              </button>
+              <button className="AddReReply" onClick={this.reReply}>
+                답글달기
+              </button>
+              <button className="ReplyDelite" onClick={this.removeReply}>
+                삭제
+              </button>
+            </footer>
+          </article>
+        </li>
+        <div className={this.state.showlist === true ? "hide" : "show"}>
```

Comment:
```suggestion
        <div className={this.state.showlist ? "hide" : "show"}>
```
true일 경우를 생략해줘도 무방할 것 같습니다!

Reply:
수정했습니다!!